app/routes/goals.py

```diff
@@ -0,0 +1,88 @@
+import os
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+from .tasks import validate_model
```

Comment:
This woks fine to import `validate_model` here from tasks, but it should be moved to and imported from a distinct file, for better organization, such as helpers.py or something like that.